Automattic/jetpack
# Global Styles on Personal: Added feature check to marketing bar

Fixes:
- https://github.com/Automattic/dotcom-forge/issues/9520

## Proposed changes:

* Added a feature check that shows the correct plan based on if the feature is enabled or not.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Make sure you have the `GLOBAL_STYLES_ON_PERSONAL_PLAN` feature enabled on your Sandbox
* Create a free site
* Add some style variations or something GS-related
* Visit the homepage, you should see the marketing banner being displayed.
* It should reference the Personal plan and link to the Personal plan checkout.
* Make sure there are no regressions on an Atomic site.

![Screenshot 2024-11-06 at 15 08 05](https://github.com/user-attachments/assets/eb618baf-d949-4100-9f5a-2b2c8c4ce0a2)
